src/sdkUtils/bridgeFromEVM.ts

```diff
@@ -442,6 +578,19 @@ async function bridgeFromEVM_toEVM(
   })
 }
 
+async function bridgeFromEVM_toTron(
+  ctx: SDKGlobalContext,
+  info: Omit<
+    BridgeFromEVMInput,
+    "fromChain" | "toChain" | "fromToken" | "toToken"
+  > &
+    KnownRoute_FromEVM_ToTron,
+): Promise<BridgeFromEVMOutput> {
+  throw new Error(
```

Comment:
Please change the `Error` to `UnsupportedBridgeRouteError`

https://github.com/Brotocol-xyz/bro-sdk/pull/23#discussion_r2144619409